src/Main.ts

```diff
@@ -153,7 +153,7 @@ export class Main {
         }
 
         let bridgeStores = {};
-        const usingNeDB = config.db === undefined;
+        const usingNeDB = config.db === undefined || config.db?.engine === "nedb";
```

Comment:
@Half-Shot Explicitly setting `config.db.engine` to `"nedb"` never worked, did it?

Also, do you agree with this change to format the sample file like Synapses sample with a lot more spacing for comments? Anoa suggested it and I like it.

Reply:
>Also, do you agree with this change to format the sample file like Synapses sample with a lot more spacing for comments? Anoa suggested it and I like it.

Spacing never hurt anyone, I'm all for it.

>Explicitly setting config.db.engine to "nedb" never worked, did it?

It should do?

Reply:
The comments in the sample config and the schema list "nedb" as a valid value.